src/test/java/com/google/sps/CommentServiceTest.java

```diff
@@ -0,0 +1,179 @@
+package com.google.sps;
+
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.googleapis.json.GoogleJsonError;
+import com.google.api.client.googleapis.json.GoogleJsonError.ErrorInfo;
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.Comment;
+import com.google.api.services.youtube.model.CommentSnippet;
+import com.google.api.services.youtube.model.CommentThread;
+import com.google.api.services.youtube.model.CommentThreadSnippet;
+import com.google.api.services.youtube.model.CommentThreadListResponse;
+import com.google.sps.servlets.CommentService;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Answers;
+import org.mockito.InjectMocks;
+import org.mockito.Matchers;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+
+public final class CommentServiceTest {
+
+  String VIDEO_ID = ""; // Arbitrary video ID, never used.
+
+  @Mock CommentThreadListResponse responseMock;
+  @Mock(answer = Answers.RETURNS_DEEP_STUBS) YouTube youtubeServiceMock;
+  @Mock(answer = Answers.RETURNS_DEEP_STUBS) YouTube.CommentThreads.List requestMock;
+  @Mock(answer = Answers.RETURNS_DEEP_STUBS) GoogleJsonResponseException exceptionMock;
+
+  @InjectMocks CommentService commentService;
+
+  @Before
+  public void setUp() throws IOException {
+    when(youtubeServiceMock.commentThreads().list(any())).thenReturn(requestMock);
```

Comment:
Nit: Remove extra line.

Reply:
I think this is just github's line wrapping lol